@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -102,4 +102,24 @@ export default {
       reset: "Tilbakestill tidsperspektiv",
     },
   },
+  DatePicker: {
+    close: "Lukk", // TODO: Consider making global
+    chooseDate: "Velg dato",
+    chooseDates: "Velg datoer",
+    chooseDateRange: "Velg start- og sluttdato",
+    chooseMonth: "Velg måned",
+    week: "Uke",
+    weekNumber: "Uke {week}",
+    selectWeekNumber: "Velg uke {week}",
+    month: "Måned",
+    nextMonth: "Gå til neste måned",
+    previousMonth: "Gå til forrige måned",
+    year: "År",
```

Comment:
More of a general question, but the keys `day`, `month` and `year` are quite generic. Would it make sense to have a "global" date key on the same level as "global"?
```
"date": {
year...
...
}
```

Reply:
Not sure. I want to be restrictive about making keys global since it complicates tings a little. (Have to `useI18n()` multiple times, and makes the `translations` prop on the component more complicated.)

Can you think of any benefits to this, other than that it avoids breaking changes if we need to make them global later?

(Side note: I wonder how hard it would be to support multiple "component names" in `useI18n()`...)

Reply:
See no real benefits other than that if felt a little off that you define how "year" is written inside datepicker, so works fine as is for me 👍